site-src/api-overview.md

```diff
@@ -1,13 +1,16 @@
-## API overview
+# Api Object Overview
 
-Prior to the AdminNetworkPolicy API there was no native tooling for Cluster Admins
-to apply security rules in a cluster-wide manner, and in some cases Network Policies
-were being incorrectly used to do so, creating a complex web of objects to be
-maintained.
+Prior to the Network Policy API the original NetworkPolicy Resource was the only
+way for k8s users to apply security rules to their kubernetes workloads. One of the
+main drawbacks to this API was that it was designed exclusively for use by the
+Application Developer, although in reality it is used by many different cluster
+personas, sometimes creating a complex web of objects to be maintained. In
+Contrast, each resource in the Network Policy API is designed to be used by a
+specific persona.
 
-With the advent of the AdminNetworkPolicy API Cluster Admins will now have the 
-ability to apply policy on in-cluster workloads with only a few simple policy 
-objects that can be applied globally.
+With the advent of the AdminNetworkPolicy and BaselineAdminNetworkPolicy
+resources Cluster Admins will now have the ability to apply policy on in-cluster
```

Comment:
nit: I know the first version only supports `in-cluster workloads` but in this part I'd keep it more generic since we mention this as a limitation already under General notes.

Reply:
Changed to just "to globally apply policy"